README.md

```diff
@@ -87,7 +88,7 @@ The slots contents are defined by
 ```
 slot_content = xor(
   msg_key,
-  Derive(recipient_key, "key_slot;prev=" + external_nonce, 32)
+  Derive(recipient_key, "key_type:key_slot", 32)
 )
```

Comment:
I'm not sure we explicitly agreed this. Are we happy with the `key_slot` being defined like this?